app/models/audit/display_item.rb

```diff
@@ -50,6 +51,45 @@ def initialize(version, users_by_id, excluded_fields = [])
     end
   end
 
+  # Class method to create multiple display items with optimized loading
+  def self.batch_create(versions, users_by_id, excluded_fields = [])
```

Comment:
Can we rename this to something like `build_batch`.  Create has a connotation that would imply it will be saving something to the database, I read `batch_create` the same as the bulk `import` method.

Since these are just "newing" up a batch, `build` feels like a more appropriate verb to me.